content/events/21-22/320_barbecue_studententuin.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Barbecue in studententuin!
+description: Zomervakantie! Barbecue, spelletjes en dan Gentse Feesten!
+created_at: xx-06-2022
+time: '16-07-2022 xx:xx'
+end: '16-07-2022 xx:xx'
+location: Voortuin van de studententuin (aan S5, Campus Sterre)
+locationlink: "Studententuin voortuin, 9000 Ghent"
+banner: "https://images0.persgroep.net/rcs/eBWqGxiUnWGoG-SXwY87Y5k2NYE/diocontent/177694046/_fill/1200/630/?appId=21791a8992982cd8da851550a453bd7f&quality=0.7"
```

Comment:
We proberen te vermijden dat we hotlinken naar images, die load je best in in pics.zeus.gent